CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
   runtime error and include details about the error in the `vm_error` field of
   the receipt. Fixes issues #3154, #3328.
 
+### Changed
+
+- All occurrences of the deprecated method `u32::max_value()` (as well as `u64`
```

Comment:
This isn't the type of change that needs to be recorded in the changelog in my opinion (nothing is changing externally).